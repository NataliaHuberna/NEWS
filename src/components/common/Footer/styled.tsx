import styled from "styled-components";
import {COLORS} from '../styles/colors';

export const StFooter = styled.div`
  background-color: ${COLORS.WHITE_BG};
  padding: 1.5rem 2.5rem;
  box-shadow: 1px 1px 7px 0 ${COLORS.SHADOW_BOX};
`;

export const StFooterWrapper = styled.div`
  max-width: 140rem;
  width: 100%;
  margin: 0 auto;
  text-align: center;
`;

export const StFooterCopyright = styled.strong`
  color: ${COLORS.GREY_TEXT};
`;
