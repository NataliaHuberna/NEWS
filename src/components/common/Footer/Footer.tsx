import React from 'react';
import {StFooter, StFooterCopyright, StFooterWrapper} from "./styled";

const Footer: React.FC = () => (
    <StFooter>
        <StFooterWrapper>
            <StFooterCopyright>
                All Rights Reserved, {new Date().getFullYear()}
            </StFooterCopyright>

        </StFooterWrapper>
    </StFooter>
);

export default Footer;
