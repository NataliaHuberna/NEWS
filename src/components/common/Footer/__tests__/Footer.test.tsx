import React from 'react';
import {shallow, mount} from 'enzyme';
import Footer from "../Footer";

describe('Footer test', () => {
    it('should render correctly', () => {
        const component = shallow(<Footer />);
        expect(component).toMatchSnapshot();
    });
    it('should render one div', () => {
        const component = mount(<Footer/>);
        expect(component.find('strong')).toHaveLength(1);
    });
});
