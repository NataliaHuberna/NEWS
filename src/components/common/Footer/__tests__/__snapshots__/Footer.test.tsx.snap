// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test should render correctly 1`] = `ShallowWrapper {}`;
